conf/messages.en

```diff
@@ -213,3 +207,10 @@ whatIsGIIN.checkYourAnswersLabel = whatIsGIIN
 whatIsGIIN.error.required = Enter whatIsGIIN
 whatIsGIIN.error.length = WhatIsGIIN must be 100 characters or less
 whatIsGIIN.change.hidden = WhatIsGIIN
+
```

Comment:
think we are just missing the **secondContact.change.hidden** used by the _row_ function in the summary file (used by the check your answers page)